File: if else/categoryandpercent .java
/* #write a program to accept percentage and display the category according to the following criteria
#<40     failed
#>=40 and <55          fair
#>=65  and  <65       good
#>=65   excellent*/

import java.util.Scanner;

class category{
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter your percentage:");
        int percent=sc.nextInt();

        if(percent<40){
            System.out.println("You have failed");
        }
        else if(percent>=40 && percent<55){
            System.out.println("Your category is fair");
        }
        else if(percent>=55 && percent<65){
            System.out.println("Your category is good");
        }
        else{
            System.out.println("Your category is excellent");
        }
    }
}

File: cpp/if else/categoryandpercent.cpp
/*write a program to accept percentage and display the category according to the following criteria
>0 and <40     failed
>=40 and <65          fair
>=65  and  <75       good
>=75 and <=100   excellent*/

#include <iostream>
using namespace std;

int main() {
    int percent;
    cout<<"Enter your percentage: ";
    cin>>percent;
    
    if(percent>=0 && percent<40){
        cout<<"You are failed";
    }
    
    else if(percent>=40 && percent<65){
        cout<<"Fair";
    }
    
    else if(percent>=65 && percent<75){
        cout<<"Good";
    }
    
    else if (percent>=75 && percent<=100){
        cout<<"Excellent";
    }
    
    else{
        cout<<"invalid input";
    }
}